### ISSUE TITLE
[BUG] Supressed ETB abilities are not trigged from Blink/Flicker effects

### ISSUE THREAD

**Issue by Zncon** [magefree/mage#9839]
When a card with an ETB such at Cloudblazer or Wall of Omens is enchanted with an ability suppression effect from Lignify, Frogify, or Darksteel Mutation, blinking/flickering the creatures causes the enchantment to correctly drop off, but the ETB does not trigger.

Rule 400.7 should mean the creature reenters as a new instance with no connection to it's previously enchanted state.

**Comment by jeffwadsworth** [comment]
[[Cloudblazer]]  [[Wall of Omens]]  [[Lignify]] [[Frogify]] [[Darksteel Mutation]]

**Comment by xenohedron** [comment]
Yeah, this fails for `ExileThenReturnTargetEffect` (e.g. Cloudshift) although it does pass for `ExileReturnBattlefieldNextEndStepTargetEffect` (e.g. Turn to Mist).

The effects from `BecomesCreatureAttachedEffect` that remove abilities and change subtypes don't appear to get cleaned up in time.

I thought `game.getState.processAction(game)` would account for that. Needs more research.
